src/odemis/acq/stream/_sync.py

```diff
@@ -1180,6 +1181,9 @@ def _acquireImage(self, n, pol_pos, px_idx, px_time, sem_time, sub_pxs,
             ccd_data = self._acq_data[self._ccd_idx][-1]
             ccd_data.metadata[MD_POS] = cor_pos
 
+            # TODO now only added to metadata if polarimetry hardware present
```

Comment:
I think that's fine. Especially, for the other type of CCD acquisitions which never expect a polarization analyser. Also, that forces us to properly handle (old) data without any MD_POL_MODE defined.